wpstarter/templates/.env.example

```diff
@@ -227,12 +227,10 @@ WP_HOME=
 #WP_PROXY_USERNAME
 #WP_PROXY_PASSWORD
 #WP_PROXY_BYPASS_HOSTS
-#WP_ACCESSIBLE_HOSTS
-#WP_HTTP_BLOCK_EXTERNAL
 
 
 #--------------------------------------------------------------------------------------------------#
 #                                          MISCELLANEOUS                                           #
 #--------------------------------------------------------------------------------------------------#
 #WP_MAIL_INTERVAL=300
-#WP_DEFAULT_THEME=twentyfifteen
\ No newline at end of file
+#WP_DEFAULT_THEME=twentyfifteen
```

Comment:
I meant, should the WPStarter define `twentysixteen` as the commented-out default theme, as presumably, that's what WP 4.4 defines as the default theme?

```php
#WP_DEFAULT_THEME=twentysixteen
```